docs/conf/portland/2019/news/thanks-recap.rst

```diff
@@ -0,0 +1,93 @@
+see our awesome community gather out in the physical world.
+
+Videos and photos
+=================
+
+Videos of all the talks are now available `in our video
+archive <https://www.writethedocs.org/videos/portland/2019/>`__. Thanks 
+to `Backpedal <https://backpedal.tv/>`__ for doing such a great job with these. 
+If you gave a lightning talk, we're working to get
+them on our site, but you can see them on 
+`YouTube <https://www.youtube.com/playlist?list=PLZAeFn6dfHpmuHCu5qsIkmp9H5jFD-xq->`__ 
+for now. (You can also subscribe to our `YouTube channel <https://www.youtube.com/channel/UCr019846MitZUEhc6apDdcQ>`_ 
+so you don't miss any future content!)
+
+We also have photos from the event available on `Flickr <https://www.flickr.com/photos/writethedocs/albums/72157691507514803>`_, 
+thanks to our wonderful photographer `Kay <https://twitter.com/goatlady>`_! 
+We really love the photos each year, and you're welcome to use them for 
+non-commercial purposes, as per our `Creative Commons License <https://creativecommons.org/licenses/by-nc-sa/2.0/>`_.
+
+Code of Conduct transparency report
+===================================
+
+As with any Write the Docs event, this conference was covered by our `community Code of Conduct <https://www.writethedocs.org/code-of-conduct/>`__.
+We aim to be as transparent with CoC incidents and enforcement as we can.
+
+To ensure that our CoC was visible and accessible, we took a number of specific steps:
+
+- Every participant agreed to the Code of Conduct in advance during registration.
+- The CoC, or a summary, was included on the conference website, conference booklet, and in posters around the conference venue.
+- The CoC was repeatedly mentioned in announcements on the stage.
+- We specifically stressed that the CoC applies to all conference spaces including the social event.
+- We encouraged all attendees to report any incident, even if they were not sure whether it was a violation.
+
+None of our organizers were approached about any CoC concerns at this event.
+We know that this is, in part, a reflection of the thoughtful and inclusive community that we're so lucky to have.
+
+That being said, we're also very conscious of the fact that just because nothing was reported doesn't mean nothing happened. If there was any personal conduct or other aspect of the conference experience that did not meet the expectations of behavior set out in the CoC, please email us at `conduct@writethedocs.org <mailto:conduct@writethedocs.org>`_.
+
+Ready for more?
+===============
+
+Don’t forget our `Vilnius conference <http://www.writethedocs.org/conf/vilnius/2019/>`_ next week,
+`Prague conference <https://www.writethedocs.org/conf/prague/2019/>`__ again in
+September, and our `Australia conference <https://www.writethedocs.org/conf/australia/2019/>`__ in
+November. Tickets are on sale for Prague. The CFP and tickets for the
+Australia conference will open soon.
```

Comment:
Should we start saying which will be live streamed?

Reply:
I think we're fine for now. Maybe in the future.